data/index.js

```diff
@@ -1,17 +1,17 @@
-const { Client } = require('pg');
+const { Pool } = require('pg');
 
-const client = new Client({
+const pool = new Pool({
   // user: '',
   host: 'localhost',
   database: 'agent_contact',
   // password: '',
   // port: 3211,
 });
```

Comment:
What's your pool's max size? This parameter can be fine tuned to enhance your performance